ocaml/xapi/xapi_pool.ml

```diff
@@ -651,6 +689,11 @@ let update_non_vm_metadata ~__context ~rpc ~session_id =
 	let (_ : API.ref_PIF option list) =
 		List.map (protect_exn (create_or_get_pif_on_master __context rpc session_id)) my_pifs in
 
+	(* update PVS sites *)
+	let my_pvs_sites = Db.PVS_site.get_all_records ~__context in
```

Comment:
Since you are using `my_pvs_sites` only once, I'd use: 

    let () = Db.PVS_site.get_all_records ~__context
      |> List.map (protect_exn (create_or_get_pvs_site_on_master __context rpc session_id))
      |> ignore